Makefile-gpu.Inc

```diff
@@ -73,6 +73,10 @@ $(BINDIR_GPU_SETUP_LIBS): $(PROJ_DIR)$(GPU_SETUP_ZIP)
 $(PROJ_DIR)$(GPU_SETUP_ZIP):
 	curl -L $(GPU_SETUP_URL_ZIP) -o $(PROJ_DIR)$(GPU_SETUP_ZIP)
 
+go-env-test: get-gpu-setup
+	go env -w CGO_LDFLAGS="$(CGO_TEST_LDFLAGS)"
+.PHONY: go-env-test
+
```

Comment:
@piersy @fasmat 
moving this target from -svm to -gpu as goland requires it to be in the same location as get-gpu-setup. 
(and i didn't want to spend time to investigate goland configuration further)

Reply:
Hey @countvonzero, I just removed that target from `Makefile-gpu.Inc` because I thought that the `print-env` and `print-env-test` targets should be sufficient. Is `go-env-test` a special target that is depended upon by goland?

Reply:
@piersy you are right. i just verified that `print-test-env` is equivalent to `go-env-test` for goland.

i think files under `https://github.com/spacemeshos/go-spacemesh/tree/develop/.run` defined the targets for goland, but to be honest i don't think we need .run directory. it seems IDE specific and checked in by accident.